build/images/Dockerfile.build.controller.ubuntu

```diff
@@ -0,0 +1,46 @@
+# Copyright 2024 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG GO_VERSION
+ARG BUILD_TAG
+FROM golang:${GO_VERSION} as antrea-build
+
+WORKDIR /antrea
+
+COPY go.mod /antrea/go.mod
+
+RUN go mod download
+
+COPY . /antrea
+
+RUN make antrea-controller
+# Disable CGO for antctl in case it is copied outside of the container image. It
+# also reduces the size of the binary and aligns with how we distribute antctl
+# in release assets.
+RUN CGO_ENABLED=0 make antctl-linux
+RUN mv bin/antctl-linux bin/antctl
+
+FROM ubuntu:22.04
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="The Docker image to deploy the Antrea CNI."
+
+USER root
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends openssl
```

Comment:
where is this coming from? It wasn't in the original unified image AFAIK. The command is also not optimized for space.

Reply:
it was a part of ovs image that we used to build the unified image, because openssl command is required on the antrea-controller-pod or else the `TestAntreaApiserverTLSConfig/ControllerApiserver` will fail.

Reply:
Just because a test currently requires the openssl command line tool is not a good enough reason to include it in the image. I suggest that you first open a PR to remove this dependency in the test. Maybe rather that exec into the controller Pod to run the `openssl` command, the test could run the command from a separate test Pod.

Reply:
done